src/i965_encoder.c

```diff
@@ -1485,5 +1494,17 @@ gen8_enc_hw_context_init(VADriverContextP ctx, struct object_config *obj_config)
 struct hw_context *
 gen9_enc_hw_context_init(VADriverContextP ctx, struct object_config *obj_config)
 {
-    return intel_enc_hw_context_init(ctx, obj_config, gen9_vme_context_init, gen9_mfc_context_init);
+    struct i965_driver_data *i965 = i965_driver_data(ctx);
+    switch (obj_config->profile){
+    case VAProfileH264ConstrainedBaseline:
+    case VAProfileH264Main:
+    case VAProfileH264High:
```

Comment:
Miss VAProfileH264MultiviewHigh and VAProfileH264StereoHigh, and you can use encoder_context->codec as conditional.